src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -18,9 +18,11 @@ public class HelpWindow extends UiPart<Stage> {
     // TODO: Remove hardcoding of message, find a way to pass in the help
     public static final String USERGUIDE_URL = "https://ay2425s1-cs2103t-t10-3.github.io/tp/UserGuide.html";
     public static final String HELP_MESSAGE =
-            "Add a person: add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]\n"
+            "Add a person: add n/NAME i/STUDENT_ID p/PHONE_NUMBER e/EMAIL [t/TAG]\n"
```

Comment:
Might be better to change PHONE_NUMBER to PHONE or other way round?

So we can keep it consistent with the Edit command prompt